docs/scos/dev/technical-enhancement-integration-guides/integrating-category-cms-blocks.md

```diff
@@ -17,295 +17,184 @@ related:
+     * @return array<\Spryker\Zed\CategoryGuiExtension\Dependency\Plugin\CategoryRelationReadPluginInterface>
      */
-    protected function getCategoryFormPlugins()
+    protected function getCategoryRelationReadPlugins(): array
     {
-        return array_merge(parent::getCategoryFormPlugins(), [
-			...
-            new CategoryFormPlugin()
-        ]);
+        return [
+            new CmsBlockCategoryRelationReadPlugin(),
+        ];
     }
 }
 ```
 
-13. Optional: To show which categories are assigned to a block on a block view page, register the category list plugin by adding `CmsBlockCategoryListViewPlugin` to the CMS Block GUI dependency provider:
+6. Optional: To show which categories are assigned to a block on a block view page, register the category list plugin by adding `CmsBlockCategoryListViewPlugin` to the CMS Block GUI dependency provider:
 
 ```php
 <?php
+
 namespace Pyz\Zed\CmsBlockGui;
 
-...
+use Spryker\Zed\CmsBlockCategoryConnector\Communication\Plugin\CmsBlockCategoryListViewPlugin;
+use Spryker\Zed\CmsBlockGui\CmsBlockGuiDependencyProvider as SprykerCmsBlockGuiDependencyProvider;
 
-class CmsBlockGuiDependencyProvider extends CmsBlockGuiCmsBlockGuiDependencyProvider
+class CmsBlockGuiDependencyProvider extends SprykerCmsBlockGuiDependencyProvider
 {
-	...
-
-	/**
-	* @return array
-	*/
-	protected function getCmsBlockViewPlugins()
-	{
-		return array_merge(parent::getCmsBlockViewPlugins(), [
-			new CmsBlockCategoryListViewPlugin(),
-		]);
-	}
+    /**
+     * @return array<\Spryker\Zed\CmsBlockGui\Communication\Plugin\CmsBlockViewPluginInterface>
+     */
+    protected function getCmsBlockViewPlugins(): array
+    {
+        return array_merge(parent::getCmsBlockViewPlugins(), [
+            new CmsBlockCategoryListViewPlugin(),
+        ]);
+    }
 }
 ```
 
 **To configure the block:**
 
-1. In the Zed UI, go to the CMS section  and navigate to the blocks section.
-2. Click Create CMS Block to create a new block.
-3. From the template drop-down, select the new template and  name the new block.
-4. Set the "Category" and enter the category URL in the Category field. While typing, the product  search will offer suggestions from the product list.
-5. View on a CMS Block edit page.
-6. View on a Category edit page.
-7. Set the block to active to use it straight away.
-8. After clicking **Save**, you will be prompted to provide glossary keys for the placeholders  included in the Twig template.
-9. Embed the block into the category page by adding the following code in the `catalog.twig` template:
+1. In the Backoffice, go to the Content section and navigate to the Blocks section.
+2. Click Create Block to create a new block.
+3. From the template drop-down, select the "Title and description block" template and name the new block.
+4. Select the Categories in the "Categories: Top", "Categories: Middle" and "Categories: Bottom" fields. While typing, the product search will offer suggestions from the product list.
```

Comment:
Does it offer suggestions from the product or from the category list? As I understand, we are assigning a cms block to categories